README.md

```diff
@@ -27,6 +27,26 @@ For the latest updates of FlarePredict refer to [https://github.com/deepsuncode/
 
 ## Installation on local machine
 
+Create a virtual environment.
+
+python -m pip virtualenv < env name >
+
+Using Pip
+
+* pip install pandas
+* python -m pip install tensorflow-macos
+* python -c 'import tensorflow as tf ;print(tf.version__)'
+* pip install scikit-learn
+* pip install keras
+
+Using conda
+* Install miniconda first 
+* eval "$(< conda path > shell.zsh hook)"
```

Comment:
Windows doesn't have eval our of the box.
please make it generic if you want to provide installation commands.

Reply:
you may also want to change the requirements in the table, tensorflow vs tensorflow-macos